tests/test_config.py

```diff
@@ -0,0 +1,42 @@
+# This file is part of the github2jira project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2021 Red Hat, Inc.
+#
+
+import os
+
+from unittest import mock
+import pytest
+
+import github2jira.githublib as githublib
+
+
+def mockenv(**envvars):
+    return mock.patch.dict(os.environ, envvars)
+
+
+@mockenv(
+    GITHUB_TOKEN="dummy",
+    GITHUB_OWNER="owner",
+    GITHUB_REPO="repo",
+    GITHUB_LABEL="sig/network",
+)
+def test_config_load():
+    githublib.config()
+
+
+def test_negative_config_load():
```

Comment:
negative? Please express what you are testing explicitly. By reading this test, I just see you expect it to raise, but I do not know why.